src/neo/SmartContract/Helper.cs

```diff
@@ -129,7 +129,7 @@ public static UInt160 ToScriptHash(this ReadOnlySpan<byte> script)
             return new UInt160(Crypto.Hash160(script));
         }
 
-        internal static bool VerifyWitnesses(this IVerifiable verifiable, StoreView snapshot, long gas)
+        internal static bool VerifyWitnesses(this IVerifiable verifiable, StoreView snapshot, long gas, WitnessVerifyStrategy strategy = WitnessVerifyStrategy.All)
```

Comment:
We don't need `WitnessVerifyStrategy` , `VerifyWitnessStateIndependent` and `VerifyWitnessStateDependent`, we already have `tx.VerifyStateIndependent` and `tx.VerifyStateDependent`, as we can tell the different type witnesses now.

eg:
```cs
tx.VerifyStateIndependent(){
     for(witness ws in tx.witness)
           if(ws.isStandardWitness)
                ws.verify()
      // ........

}

tx.VerifyStateDependent(){
     for(witness ws in tx.witness)
           if(!ws.IsStandardWitness)
                ws.verify()
      // ........

}
```